Exo-donut-dix-novembre/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Exo-donut-dix-novembre",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "npm run gulp",
+    "gulp": "gulp"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "d3-shape": "^1.0.3",
+    "moment": "^2.16.0",
+    "rollupify": "^0.3.5"
+  },
+  "devDependencies": {
+    "babel-preset-es2015": "6.6.0",
+    "babelify": "7.2.0",
+    "browserify": "13.0.0",
+    "gulp": "3.9.1",
```

Comment:
gulp shouldn't be necessary, try not using it next time.
Reminder: https://github.com/DavidBruant/contenu-formations-web/pull/7#discussion_r87390255 ;-)